UnityProject/Assets/Scripts/Closets/ClosetControl.cs

```diff
@@ -490,7 +578,7 @@ private void OpenItemHandling()
 			}
 			else
 			{
-//				netTransform.AppearAtPositionServer(pos);
+				//				netTransform.AppearAtPositionServer(pos);
```

Comment:
kill comments!

Reply:
Ill remove this, this was legacy, not sure when I changed the spacing...